packages/pods-core/src/v2/pods/export/GoogleDocsExportPodV2.ts

```diff
@@ -0,0 +1,138 @@
+import {
+  axios,
+  DendronError,
+  DEngineClient,
+  DVault,
+  NoteProps,
+  ResponseUtil,
+  RespV2,
+  Time,
+} from "@dendronhq/common-all";
+import { JSONSchemaType } from "ajv";
+import FormData from "form-data";
+import _ from "lodash";
+import {
+  ConfigFileUtils,
+  ExportPodV2,
+  GoogleDocsV2PodConfig,
+  HTMLPublishPod,
+  PodUtils,
+  RunnableGoogleDocsV2PodConfig,
+} from "../../..";
+
+export type GoogleDocsExportReturnType = RespV2<{
+  /**
+   * Document Id of the exported Note
+   */
+  documentId?: string;
+}>;
+
+/**
+ * GDoc Export Pod (V2 - for compatibility with Pod V2 workflow). Supports
+ * both exportText() and exportNote()
```

Comment:
update comment - doesn't look like we support `exportText()` yet?

Reply:
Yes, will update this.